JSQMessagesViewController/Factories/JSQMessagesBubbleImageFactory.m

```diff
@@ -49,14 +49,15 @@ - (instancetype)initWithBubbleImage:(UIImage *)bubbleImage capInsets:(UIEdgeInse
         else {
             _capInsets = capInsets;
         }
+        
+        _isRightToLeftLanguage = (UIUserInterfaceLayoutDirectionRightToLeft == layoutDirection);
```

Comment:
`isRightToLeftLanguage` can become a `readonly` computed property for convenience